### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by sudswasavi**
This PR, modifies the helm charts to become more readable.
Plus makes the controller version auto-discoverable.

It also implements a slow retry layer in AKO.
The slow retry layer is used only if there's a problem in contacting the Avi Controller. 
We now call AviRestOperate inside a go routine and timeout post 80 seconds.

### CODE REVIEW

File: tests/bootuptests/stale_obj_delete_test.go [hunk 1]
```diff
@@ -116,9 +116,12 @@ func injectMWForObjDeletion() {
 		} else if r.Method == "GET" {
 			integrationtest.FeedMockCollectionData(w, r, mockFilePath)
 
-		} else if strings.Contains(url, "login") {
+		} else if strings.Contains(url, "initial-data") {
 			w.WriteHeader(http.StatusOK)
-			w.Write([]byte(`{"success": "true"}`))
+			w.Write([]byte(`{"version": {"Version": "20.1.2"}}`))
+		} else if strings.Contains(url, "initial-data") {
```
Comment (monotosh-avi) [inline tests/bootuptests/stale_obj_delete_test.go:122]: repeated checks ?

Comment (sudswasavi) [reply tests/bootuptests/stale_obj_delete_test.go:118]: Great catch. Done.

File: internal/rest/dequeue_nodes.go [hunk 2]
```diff
@@ -399,21 +399,23 @@ func (rest *RestOperations) ExecuteRestAndPopulateCache(rest_ops []*utils.RestOp
 									utils.AviLog.Infof("key: %s, msg: Error is not of type AviError, err: %v, %T", key, rest_ops[i].Err, rest_ops[i].Err)
 									continue
 								}
-								retryable, fastRetryable := rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
-								fastRetry = fastRetry || fastRetryable
-								retry = retry || retryable
+								regularRetry, slowRetry = rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
```
Comment (monotosh-avi) [inline internal/rest/dequeue_nodes.go:402]: RefreshCacheForRetryLayer would return true always for regular retry, so we would never go to slow retry.

Comment (sudswasavi) [reply internal/rest/dequeue_nodes.go]: Retained the logic that was there earlier and handling the slow retry checks separately.

**Review by parimanur — APPROVED** [review]

**Review by monotosh-avi — APPROVED** [review]

**Pull request merged**